content/ko/docs/tasks/configmap-secret/managing-secret-using-kustomize.md

```diff
@@ -0,0 +1,140 @@
+---
+title: kustomize를 사용하여 시크릿 관리
+content_type: task
+weight: 30
+description: kustomization.yaml 파일을 사용하여 시크릿 오브젝트 생성.
+---
+
+<!-- overview -->
+
+쿠버네티스 v1.14부터 `kubectl`은 다음을 지원한다.
+[Kustomize를 이용한 쿠버네티스 오브젝트의 선언형 관리](/ko/docs/tasks/manage-kubernetes-objects/kustomization/).
```

Comment:
[[원문](https://raw.githubusercontent.com/kubernetes/website/main/content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md)]
> Since Kubernetes v1.14, `kubectl` supports
[managing objects using Kustomize](/docs/tasks/manage-kubernetes-objects/kustomization/).

```suggestion
쿠버네티스 v1.14부터 `kubectl`은 
[Kustomize를 이용한 쿠버네티스 오브젝트의 선언형 관리](/ko/docs/tasks/manage-kubernetes-objects/kustomization/)를 지원한다.
```